### PR TITLE
Updates README.rst

### PR DESCRIPTION
**Pull request by alex4200**
Formatting changes

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -127,57 +125,57 @@ All the config files are working for both the 'post-synaptic cell only' and the
...
 Run the simulation using hoc
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can also run the simulation using hoc. In order to do that, you will have to first create the hoc files with the following line:
+You can also run the simulation using hoc. In order to do that, you will have to first create the hoc files with the following line::
 
     python create_hoc.py --config_path config_path
 
-Where config_path is the path to a specific config file. You will find the available config files in the config folder.
+Where ``config_pat``h is the path to a specific config file. You will find the available config files in the config folder.
```
Comment (wvangeit) [inline README.rst:175]: There is a type in config_path

File: README.rst [hunk 2]
```diff
@@ -11,29 +11,27 @@ EModelRunner is a python library designed to run the cell models provided by the
...
 Installing EModelRunner
 =======================
 
-The usual way to install EModelRunner is using pip.
-
-In that case, you probably want to use a python virtual environment.
+The usual way to install EModelRunner is using pip. In that case, you probably want to use a python virtual environment.
 
-Pip install emodelrunner from PyPi will be available after open sourcing::
+Instll uing ``pip``::
```
Comment (wvangeit) [inline README.rst:34]: Also a small type here

File: README.rst (hunk 1 above)
Comment (alex4200) [reply README.rst]: thanks for noticing, should be fixed now

File: README.rst (hunk 2 above)
Comment (alex4200) [reply README.rst]: thanks for noticing, should be fixed now

**Review by wvangeit — APPROVED** [review]

**Review by AurelienJaquier — APPROVED** [review]

**Pull request merged**